src/main/java/com/checkmarx/flow/dto/ScanRequest.java

```diff
@@ -80,6 +81,7 @@
     private Map<String, String> additionalMetadata;
     private List<VulnerabilityScanner> vulnerabilityScanners;
     private ScaConfig scaConfig;
+    private ASTConfig astConfig;
```

Comment:
I think we need to strive to define them all in Cx-Flow

Reply:
I think we strive to implement everything we can inside Cx-Flow, the reason ScaConfig is in the SDK's side is because it part on the SCA results logic implementation, whereas ASTConfig isn't.